backend/requirements.txt

```diff
@@ -13,6 +13,6 @@ pyexcel-ods>=0.5.2
 gevent~=1.3.5
 python-dateutil>=2.7.0
 setuptools>=40.0.0
--e git+https://bitbucket.org/magenta-aps/flask_saml@development#egg=flask_saml
+-e git+https://github.com/magenta-aps/python3-saml@attribute_fix#egg=python3-saml
```

Comment:
What modifications do we have for `python3-saml`?

Reply:
https://github.com/magenta-aps/python3-saml/commit/5277d78cf37b56979b0c2778b3e5589122f5e922

Reply:
Reading the relevant bug, it seems to have security repercussions — what triggers this?